packages/react-widgets/src/widgets/BarWidget.js

```diff
@@ -156,6 +156,7 @@ function BarWidget({
             height={height}
             animation={animation}
             filterable={filterable}
+            isLoading={isLoading}
```

Comment:
Why is this injected externally? Shouldn't this be just a prop that the 'widget' launching the query, passes down to the UI?

Reply:
This is what I'm doing here, isn't it? passing down the prop from the parent to the UI, as in line 137 for the WrapperWidgetUI.

The logic was already there, but the UI wasn't receiving it.

Reply:
Yes, misunderstanding on my side. It's fine